CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.0-beta.3 (2021-10-16)
+
+- Updated peer dependencies. [#62](https://github.com/blackbaud/skyux-avatar/pull/62)
```

Comment:
```suggestion
- Updated peer dependencies. [#64](https://github.com/blackbaud/skyux-avatar/pull/64)
```
This is currently linking to a PR that was closed, not merged. I'm pretty sure we are updating the dependencies in this PR and just need the link to point back to this PR.